yaml/overlays/eks/eks-config.yaml

```diff
@@ -7,7 +7,7 @@ data:  # Configuration template for VPP in EKS
   service_prefix: 10.100.0.0/16
   vpp_dataplane_interface: eth0
   veth_mtu: "1410"
```

Comment:
Should we just leave the mtu blank now that we have better autodetection logic?

Reply:
Hmmm, we \*could\* do this, but let's check if things work fine with a 9001 mtu then 😄